projects/qcms/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER pdknsk@gmail.com
+RUN apt-get update && apt-get install -y wget unzip
+RUN wget -qO qcms.zip \
+    https://hg.mozilla.org/mozilla-central/archive/tip.zip/gfx/qcms/ && \
```

Comment:
why not use hg? 
I am worried that downloading a tip archive may cause some problems (e.g. oss-fuzz won't see the revision numbers)

Reply:
I assume this was to save time on clones; the spidermonkey target uses the github mirror in order to enable shallow clones.

Reply:
Yep, that was the idea, but I can change it to use `hg`.